### ISSUE TITLE
Magnet links are not working for the Piratebay when shields are up.

### ISSUE THREAD

**Issue by Zerophase** [brave/brave-browser#4185]
## Description 
Clicking on a magnet link in Linux does not open the link in the selected torrent client. You cannot even click on the link to copy it into the torrent client.

## Steps to Reproduce
   1. Open Brave on Linux
   2. Navigate to: https://www.thepiratebay.org/torrent/11722376/Zeiram
   3. Click on the Link

## Actual result:
Nothing happens.

## Expected result:
system torrent client or brave torrent tab opens

## Reproduces how often: 
Every time.

## Brave version (brave://version info)
0.62.51-2

## Version/Channel Information:
No idea.

- Can you reproduce this issue with the current release?
Yes.

- Does the issue resolve itself when disabling Brave Shields?
Resolves when Brave Shields is disabled.

**Comment by srirambv** [comment]
Issue is with the anchor element somewhere at the end of the text. Works fine https://youtu.be/iQnE3-eRsW4

Closing as wont fix. This works just fine on Brave.

**Comment by Zerophase** [comment]
@srirambv Yeah, there is a small spot in the bottom right corner of "magnet link" you can click on with  careful aim.  Isn't this an area that could improve ad blocking detection, but take substantial work to actually address the general case?

**Comment by srirambv** [comment]
Further investigating this isn't related to FP. Reopening to recheck again

**Comment by bsclifton** [comment]
Closing as `wontfix`.

WebTorrent as a feature has been removed with https://github.com/brave/brave-browser/issues/46416 which goes out with version 1.81 (releasing 5th of August, 2025)

See https://github.com/brave/brave-browser/issues/46430 for more information
